examples/examples_test.go

```diff
@@ -144,6 +145,14 @@ func validateObject(obj runtime.Object) (errors field.ErrorList) {
 	return errors
 }
 
+func validateschedulerpolicy(obj runtime.Object) error {
+	switch t := obj.(type) {
+	case *schedulerapi.Policy:
+		return schedulerapivalidation.ValidatePolicy(*t)
+	default:
+		return fmt.Errorf("obj type is not schedulerapi.Policy")
+	}
+}
```

Comment:
Please add a blank line after the function.

Reply:
OK, I have re push it.Thanks